### ISSUE TITLE
 Remove the requirement of Flash Attn

### ISSUE THREAD

**Issue by zhuole1025** [kijai/ComfyUI-LuminaWrapper#2]
Hi kijai,
Thanks for your great efforts in implementing the ComfyUI version of our Lumina framework! We noticed that a lot users had trouble with installing the flash attention. Therefore, we update our code and add an argument `use_flash_attn` to allow the user to disable flash attention during training or inference. Our updated code: https://github.com/Alpha-VLLM/Lumina-T2X/tree/main/lumina_next_t2i_mini

**Comment by kijai** [comment]
Hey!

Thank you for the effort to support inference without flash_attn.

However I already tried this and it seems the fallback SDP attention code does not seem to work with the GQA -model as the query states (32) are different shape than key/value states (8), and this is not taken into account? I have limited knowledge of attention code so I don't know more. The error I get is, happens:

`RuntimeError: The size of tensor a (32) must match the size of tensor b (8) at non-singleton dimension 1`
